xblockutils/base_test.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
```

Comment:
I only did this because virtually all the methods in this file will look like this:

```python
def wait_until_exists(self, selector):
    """ Wait until selector exists """
```

I personally am not a fan of a buffer on pylint errors, so I'll just add in the docstrings.